popup.js

```diff
@@ -1,111 +1,157 @@
-//loading buttons
-var searchTextArea = document.getElementById('search');
+var searchTextArea = document.getElementById("search");
 var add = document.getElementById("add");
 var save = document.getElementById("download");
-var clear = document.getElementById('clear');
-var textarea = document.getElementById('textarea');
-var index = 0;
-var buttons = `<button class="tooltip tilebutton searchTile"><img src="https://img.icons8.com/material-outlined/15/000000/search--v2.png"/></a>
-<span class="tooltiptext">very soon!</span></button>
-<button class="tooltip tilebutton deleteTile"><img src="https://img.icons8.com/material-rounded/15/000000/delete.png"/></a>
-<span class="tooltiptext">soon!</span></button>`;
-
-
-//restoring localStorage
+var clear = document.getElementById("clear");
+var textarea = document.getElementById("textarea");
 var saved = localStorage.getItem('listItems');
-var note = localStorage.getItem('note');
+var list = document.getElementById('list');
+var index = 0;
```

Comment:
You should avoid using `var` wherever possible (it's the old way of doing things!) and work out whether you need to use `let` or `const` in future.

Rule of thumb:
* `let` for anything you can change, e.g. if you say `index = 0` and then later set `index = 1`
* `const` for anything that shouldn't change, e.g. `add = document.getElementById("add")` probably isn't going to change to point to a different element later on

Further reading: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75